vapi/libical.vapi

```diff
@@ -1464,9 +1464,9 @@ namespace ICal {
 		[CCode (cname = "icaltimezone_get_tznames")]
 		public unowned string get_tznames ();
 		[CCode (cname = "icaltimezone_get_utc_offset")]
-		public int get_utc_offset (out ICal.Time? tt, out int is_daylight);
+		public int get_utc_offset (ICal.Time? tt, out int is_daylight);
 		[CCode (cname = "icaltimezone_get_utc_offset_of_utc_time")]
-		public int get_utc_offset_of_utc_time (out ICal.Time tt, out int is_daylight);
+		public int get_utc_offset_of_utc_time (ICal.Time tt, out int is_daylight);
```

Comment:
Probably should add a comment that this has been manually patched  (lines 1467 & 1469)